src/gui/guiFormSpecMenu.h

```diff
@@ -354,6 +353,7 @@ class GUIFormSpecMenu : public GUIModalMenu
 		std::string focused_fieldname;
 		GUITable::TableOptions table_options;
 		GUITable::TableColumns table_columns;
+		IGUIElement *background_parent;
```

Comment:
```suggestion
		IGUIElement *background_parent = nullptr;
```
Needs to be initialized for security.

Reply:
Done.